test/unit/vl-form-validation.test.html

```diff
@@ -19,40 +19,41 @@
 
   <script type="module">
     import {vlFormValidation} from '../../src/vl-form-validation.js';
-    const sandbox = sinon.createSandbox();
-
-    setup((done) => {
-      Promise.all([
-        vlFormValidation(Object).awaitUntilReady(),
-      ]).then(() => done());
-    });
-
-    teardown(() => {
-      sandbox.restore();
-    });
 
     suite('vl-form-validation', () => {
-      class VlFormValidationImplementatie extends vlFormValidation(Object) { }
+      const sandbox = sinon.createSandbox();
 
+      setup((done) => {
+        customElements.whenDefined('vl-form-validation-test').then(() => done());
+      });
+
+      teardown(() => {
+        sandbox.restore();
+      });
+    
       test('een class die de form validatie mixin toepast heeft een dress functie om de form validatie te activeren', () => {
-        const implementatie = new VlFormValidationImplementatie();
-        assert.isDefined(implementatie.dress);
+        const object = {};
+        assert.isUndefined(object.dress);
+        Object.assign(object, vlFormValidation);
+        assert.isDefined(object.dress);
       });
 
       test('de dress functie zal de form validatie activeren', () => {
         sandbox.spy(window.vl.formValidation, 'dress');
-        const implementatie = new VlFormValidationImplementatie();
+        const object = {};
+        Object.assign(object, vlFormValidation);
         const element = document.createElement('div');
-        implementatie.dress(element);
+        object.dress(element);
         assert(window.vl.formValidation.dress.calledWith(element));
```

Comment:
Stond er al, maar ik denk dat het een kwestie van stijl is om assert(...calledWith) te gebruiken ipv assert.calledWith(...)
Misschien dat in het laatste geval boodschap duidelijker is indien niet zo?

Reply:
Is inderdaad kleur en smaak, de testen zijn door verschillende developers geschreven dus zal altijd merkbaar zijn.